### DISCUSSION TITLE
NEI testdata

### DISCUSSION THREAD

**Discussion by eva0marques** [Spatiotemporal-Exposures-and-Toxicology/NRT-AP-Model#268] [General]
@dawranadeep @sigmafelix 
Is there any covariate calculation from the raw NEI data? 
If yes, which data files should I replicate (in smaller for our testdata subset)? Maybe all the `onroad_####.csv` files in the `aadt2020` folder? Thanks!

**Comment by dawranadeep** [comment]
Eva,

By "calculation", if you mean a buffer variable, then no. The data is
available at county-level.

Ranadeep

**Reply by eva0marques** [reply]
I don't know, any extraction function at monitors location for eg (or at prediction points).

**Comment by dawranadeep** [comment]
In this
<https://github.com/Spatiotemporal-Exposures-and-Toxicology/NRTAPmodel/blob/download_aadt/R/compute_NEI.R>
R
file, there is a function *compute_NEI(input_dat, onroad) *(line 65), which
extracts the NEI data at the location. Line 58 in the file has a demo
format of the *input_dat* file.

**Reply by eva0marques** [reply]
Ok thanks.
